war/src/main/js/handlebars-helpers/ifneq.js

```diff
@@ -0,0 +1,5 @@
+export default function ifneq(o1, o2, options) {
```

Comment:
Why are these functions necessary?

Reply:
I mean: they were in a another file before. But do you think there could be places where they are used from outside?

Reply:
Handlebars templates are compiled and processed by webpack. In order to do this, Handlebars needs to know which helpers are available at compile time. That's why I extracted as many helpers as possible to this dir. Note that they can still be 

Every helper that is registered at runtime must be listed on the _webpack.config.js file_. Only a small set of helpers are still registered this way, and the reason for that is that they break when extracted (due to their use of closure scope).

Reply:
Resolving this one. Feel free to unresolve it if you think further discussion is needed.